src/scripts/events/EventList.js

```diff
@@ -46,7 +46,10 @@ export const eventList=()=>{
         }else{
             let isFirst=true
             correctEvents.forEach(eventObj=>{
-                if(isFirst){
+                if(isFirst&&eventObj.userId===parseInt(sessionStorage.getItem("activeUser"))){
+                    contentTarget.innerHTML+=eventHTML(eventObj, isFirst, true)
```

Comment:
I see that we are passing three arguments into eventHTML, but eventHTML only takes one argument. I'd like to see eventHTML updated to have the same number of arguements.